config/application.yml.default

```diff
@@ -87,6 +87,7 @@ doc_auth_max_capture_attempts_before_native_camera: 3
 doc_auth_max_submission_attempts_before_native_camera: 3
 doc_auth_not_ready_section_enabled: false
 doc_auth_selfie_capture: '{"enabled":false}'
+doc_auth_sdk_capture_orientation: '{"horizontal": 100, "vertical": 0}'
```

Comment:
I am not very clear on the usage of this flag. Is it used to indicating taking the pictures with mobile device horizontal view/vertical view, or is it used to "reorienting " the picture after they have been taken?

Reply:
That's a good question! The usage would be to force the SDK capture to happen horizontally or vertically.

Currently the user continues the flow vertically, and when they get to the SDK step they have to turn their phone horizontally to capture photos, and then back to vertically to finish the process. This flag is to try testing out what the user experience is like if the SDK capture is vertical, so that users don't have to turn their phones at all.

Reply:
Do you have any concerns about this use case, @dawei-nava ?

Reply:
@night-jellyfish that's clear 👍